pkg/controller/container-runtime-config/helpers.go

```diff
@@ -876,6 +876,23 @@ func policyItemFromSpec(policy apicfgv1alpha1.Policy) (signature.PolicyRequireme
 			return nil, err
 		}
 		signedOptions = append(signedOptions, signature.PRSigstoreSignedWithFulcio(prSigstoreSignedFulcio), signature.PRSigstoreSignedWithRekorPublicKeyData(policy.RootOfTrust.FulcioCAWithRekor.RekorKeyData))
+	case apicfgv1alpha1.PKIRootOfTrust:
+		pkiOptions := []signature.PRSigstoreSignedPKIOption{}
+		pkiOptions = append(pkiOptions, signature.PRSigstoreSignedPKIWithCARootsData(policy.RootOfTrust.PKI.CertificateAuthorityRootsData))
```

Comment:
nit: instead of appending to the empty object, why not just define the initial pkiOptions directly?

Reply:
Thanks for the suggestion. I prefer to leave it as it is for now to keep consistency with the rest of the code.